docs/help.rst

```diff
@@ -23,16 +22,22 @@ Ask for help, or just say hi!
       :class-card: sd-bg-light sd-bg-text-light
       :link: https://github.com/robusta-dev/robusta/issues
 
+--------------------------------
 Commercial Support
-^^^^^^^^^^^^^^^^^^^
+--------------------------------
 Contact support@robusta.dev for details.
 
+--------------------------------
 Common Errors
-^^^^^^^^^^^^^
+--------------------------------
 
-This list contains some common errors we have encountered over time. If you can't find your answer here,
-contact support@robusta.dev or go to `our slack channel <https://bit.ly/robusta-slack>`_.
+This list contains some common errors we have encountered over time.
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Robusta CLI tool
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Errors installing the robusta cli config creation tool. Not relevant when installing via browser at platform.robusta.dev.
```

Comment:
Maybe use the same term we use on the installation guide? (Web Installation)
`Not relevant when using the Web Installation method`